src/vmm/src/builder.rs

```diff
@@ -702,7 +704,11 @@ fn attach_net_devices(
         event_manager
             .add_subscriber(net_device.clone())
             .map_err(RegisterEvent)?;
-        let id = net_device.lock().unwrap().id().clone();
+        let id = net_device
+            .lock()
+            .expect("Failed to get net device due to poisoned lock")
```

Comment:
nit: to save some binary size, I would use a standard common message `Poisoned lock` for all `lock()` fails.
The location of the failing lock is evident from the crash.

Reply:
I agree with this as long as all messages of this type are replaced.

Reply:
Done